lib/capistrano/tasks/stretcher.rake

```diff
@@ -100,7 +111,13 @@ namespace :stretcher do
   task :upload_tarball do
     on application_builder_roles do
       as 'root' do
-        execute :aws, :s3, :cp, "#{local_tarball_path}/current/#{fetch(:local_tarball_name)}", fetch(:stretcher_src)
+        if fetch(:stretcher_src).start_with?("s3://")
+          # upload to s3
+          execute :aws, :s3, :cp, "#{local_tarball_path}/current/#{fetch(:local_tarball_name)}", fetch(:stretcher_src)
```

Comment:
Can you unify `"#{local_tarball_path}/current/#{fetch(:local_tarball_name)}"` on L116 and L119?